delta-sharing-protocl-api-description.yml

```diff
@@ -0,0 +1,695 @@
+        '404':
+          $ref: "#/components/responses/404"
+        '500':
+          $ref: "#/components/responses/500"
+        '200':
+          description: |
+            The user could successfully list the shares
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ListShareResponse'
+
+  '/shares/{share}':
+    get:
+      operationId: 'GetShare'
+      summary: 'get the metadata of a share'
+      tags:
+        - shares
+        - official
+      parameters:
+        - in: path
+          name: share
+          required: true
+          description: 'Named share'
+          schema:
+            type: string
+      responses:
+        '400':
+          $ref: "#/components/responses/400"
+        '401':
+          $ref: "#/components/responses/401"
+        '403':
+          $ref: "#/components/responses/403"
+        '404':
+          $ref: "#/components/responses/404"
+        '500':
+          $ref: "#/components/responses/500"
+        '200':
+          description: 'The share''s metadata was successfully returned'
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Share"
+
+  '/shares/{share}/schemas':
+    get:
+      operationId: 'ListSchemas'
+      summary: 'List the schemas in a share'
+      tags:
+        - shares
+        - official
+      parameters:
+        - in: path
+          name: share
+          required: true
+          description: 'Named share to list the schemas'
+          schema:
+            type: string
+        - in: query
+          name: maxResults
+          required: false
+          description: |
+            The maximum number of results to be returned in a single page. If
+            the number of potential results exceeds the number of maximum
+            results, the response will contain a `nextpageToken` which can be
+            used in subsequent requests.
+          example: '30'
+          schema:
+            type: number
+            format: int32
+            default: 500
+        - in: query
+          name: pageToken
+          required: false
+          description: Optionally provided page token for requesting a subsequent page of results
+          schema:
+            type: string
+      responses:
+        '400':
+          $ref: "#/components/responses/400"
+        '401':
+          $ref: "#/components/responses/401"
+        '403':
+          $ref: "#/components/responses/403"
+        '404':
+          $ref: "#/components/responses/404"
+        '500':
+          $ref: "#/components/responses/500"
+        '200':
+          description: |
+            The user could successfully list the schemas in the given share
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ListSchemasResponse'
+
+  '/shares/{share}/schemas/{schema}/tables':
```

Comment:
'shares/{share}/all-tables' is missing from the doc.

Reply:
added